cluster-autoscaler/cluster_autoscaler.go

```diff
@@ -410,7 +380,7 @@ func main() {
 		kube_leaderelection.RunOrDie(kube_leaderelection.LeaderElectionConfig{
 			Lock: &resourcelock.EndpointsLock{
 				EndpointsMeta: apiv1.ObjectMeta{
-					Namespace: "kube-system",
```

Comment:
The flag will have a default value of kube-system, but yes, we will have to update exmples/k8s manifests with the next release.